src/main/modules/mpris.ts

```diff
@@ -0,0 +1,107 @@
+/**
+ * Module in charge of setting and manage mpris service for linux
+   TODO: fetch cover, handle seek bar, handle volume    
+ */
+import * as electron from 'electron';
+import ModuleWindow from './module-window';
+import { TrackModel } from '../../shared/types/interfaces';
+import { fetchCover } from '../../shared/utils';
+import * as mpris  from 'mpris-service';
+const { app, ipcMain } = electron;
+
+class MprisModule extends ModuleWindow {
+
+  player = mpris({
+      name: 'museeks',
+      identity: 'Musseks Music Player',
+      canRaise: true,
+      supportedUriSchemes: ['file'],
```

Comment:
You should try to add `data` here, it may make the dataUri cover work.

https://specifications.freedesktop.org/mpris-spec/2.0/Media_Player.html#Property:SupportedUriSchemes

Reply:
Ok. I'll try it. :+1: